ui/apps/platform/src/Components/PatternFly/CheckboxSelect.tsx

```diff
@@ -17,6 +17,7 @@ export type CheckboxSelectProps = {
     placeholderText?: string;
     toggleIcon?: ReactElement;
     toggleId?: string;
+    menuAppendTo?: () => HTMLElement;
```

Comment:
You made this optional, but you didn't supply a default value in the props destructuring in the function def below. 
Could this cause an issue at runtime?